docs/parameters.md

```diff
@@ -47,6 +47,8 @@ There are multiple use cases from the users perspective that dictate what parame
 | `--pulp-database-name` | Name of the Pulp database | `--foreman-proxy-content-pulpcore-postgresql-db-name` |
 | `--pulp-database-user` | Owner of the Pulp database | `--foreman-proxy-content-pulpcore-postgresql-user` |
 | `--pulp-database-password` | Password for Pulp database | `--foreman-proxy-content-pulpcore-postgresql-password` |
+| `--foreman-initial-admin-username` | Initial username for the admin user | `--foreman-initial-admin-username` |
+| `--foreman-initial-admin-password` | Initial password for the admin user | `--foreman-initial-admin-password` |
```

Comment:
Sorry I didn't catch this earlier, can you remove these entries from the Undetermined section?

https://github.com/theforeman/foremanctl/pull/225/files#diff-7ad93860b53d91cc603737c90612d6533ce53edf882eda504ddd476df97c9a2fR86-R87